features/recommendations.feature

```diff
@@ -7,8 +7,39 @@ Background:
     Given the following Recommendations
         | product_a_sku | product_b_sku | recommendation_type  | likes |
         | HYJtLnYf      | cUnyEDwP      | CROSS_SELL    | 0 |
+        | GQGEsdfq      | cUafQfef      | CROSS_SELL    | 0 |
+        | FQEFQrQs      | cEdasdTs      | UP_SELL       | 1 |
+        | dasdfeaQ      | FefaffeQ      | BUNDLE        | 3 |
 
 Scenario: The server is running
     When I visit the "Home Page"
     Then I should see "Recommendation RESTful Service" in the title
-    And I should not see "404 Not Found"
\ No newline at end of file
+    And I should not see "404 Not Found"
+
+Scenario: Search for HYJtLnYf
+    When I visit the "Home Page"
+    And I set the "Product A SKU" to "HYJtLnYf"
+    And I press the "Search" button
+    Then I should see the message "Success"
+    # And I should see "HYJtLnYf" in the results
```

Comment:
![image](https://github.com/CSCI-GA-2820-SP24-003/recommendations/assets/74287387/1cb19854-ced7-4d0c-ab72-39abe11e2bc8)
Indeed, nothing is returned for the "search."

Reply:
<img width="1450" alt="Screenshot 2024-04-10 at 20 23 05" src="https://github.com/CSCI-GA-2820-SP24-003/recommendations/assets/29853512/bdf79f66-6ded-44b6-8b48-271c2782197d">
When I try to click Search button on the website with only "Product A SKU" field filled, it works.